activerecord/test/cases/adapter_test.rb

```diff
@@ -442,6 +442,20 @@ def setup
       end
     end
 
+    def test_create_with_query_cache
+      @connection.enable_query_cache!
+
+      count = Event.count
+      binds = [[Event.column_for_attribute("id"), 1]]
```

Comment:
Don't use the deprecated legacy binds (an array of `[column || nil, value]`).

https://buildkite.com/rails/rails/builds/74129#f336c8d3-da13-4408-8e86-cedecdabba7d/1066-1075

```suggestion
      binds = [1]
```